src/blueapi/worker/task.py

```diff
@@ -68,8 +68,13 @@ def _lookup_params(
 
 
 @dataclass
-class ActiveTask:
-    name: str
+class TrackableTask:
+    """
+    An internal representation for use by the worker, holding a
+    task with it's ID and current status
```

Comment:
If we're saying that a Task's status is either complete (or not) and in error (or not), that agrees with my thoughts on Pause/Resume being operations on the RunEngine and not of the Task.